php/libraries/Module.class.inc

```diff
@@ -144,15 +144,6 @@ class Module
         // Hacks so that existing display() functions load the right template
         // for different page types.
         $cls->menu = $page;
-        if (!$cls->_hasAccess()) {
```

Comment:
This had to be moved to `NDB_Page->displayUserPage` in order for the `$tpl_data['console']` variable to keep working, otherwise setup gets called from NDB_Caller before the `ob_start` output buffering is setup.